ewok_village/map.conf

```
@@ -0,0 +1,14 @@
+team.1 = red
+team.1.color = red
+team.1.pos = (38,-19,39)
+team.2 = blue
+team.2.color = blue
+team.2.pos = (-38,-19,-39)
+r = 50
+rotation = z
+h = 120
+author = AnthonyDe and JediKnight
+name = Ewok Map
```

Shouldn't here be `Ewok Village`?
```suggestion
name = Ewok Village
```